docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: explain_debug
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2047
```

Comment:
yeah, 2047 should be in under RFC PR, and `Cockroach Issue: xxx` should refer to the implementation issue/pr